### DISCUSSION TITLE
FocusVisualKind Reveal in Skia/Gtk/Linux

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#12403] [Q&A] [answered]
What would it take to get the glow from the FocusVisualKind.Reveal effect in Skia (specifically Linux Framebuffer)?

**Comment by MartinZikmund** [comment, marked as answer]
It is technically possible, we just didn't get to this yet. Please create an issue and we can prioritize 👍

**Reply by HakanL** [reply]
Very well, I've added the issue: https://github.com/unoplatform/uno/issues/12413
